docs/en/reference/index.rst

```diff
@@ -0,0 +1,94 @@
+Introduction
+============
+
+The Doctrine Coding Standard is a set of PHPCS rules applied to all Doctrine projects. It is based on `PSR-1 <https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md>`_
+and `PSR-2 <https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md>`_, with some noticeable
```

Comment:
I believe this section is redundant: PSR-2 itself is based on PSR-1, so I don't think we need to point this out.